lib/merch_calendar/date_calculator.rb

```diff
@@ -49,6 +51,17 @@ def end_of_month(year, merch_month)
       end
     end
 
+    def start_of_week(year, month, merch_week)
+      weeks = MerchCalendar.weeks_for_month(year, month)
+
+      week = weeks.find { |week| week.week == merch_week }
+      week.start_of_week
+    end
+
+    def end_of_week(year, month, merch_week)
+      start_of_week(year, month, merch_week) + 6
```

Comment:
Lol, you should see the rest of this file beyond just this small diff.  One might say I was keeping consistent with the style of the code around it 😉 

But, point taken.  I can do better.  Will address.